airbyte-integrations/bases/standard-destination-test/build.gradle

```diff
@@ -5,6 +5,9 @@ dependencies {
     implementation project(':airbyte-db:db-lib')
     implementation project(':airbyte-commons-worker')
     implementation project(':airbyte-config:config-models')
+    implementation project(':airbyte-config:config-persistence')
```

Comment:
Why is this dependency required for destination tests?

Reply:
Thank you, removed this dependency

Reply:
Thank you, removed this dependency